src/lib/stepper/step-header.html

```diff
@@ -1,6 +1,8 @@
 <div class="mat-step-header-ripple" mat-ripple [matRippleTrigger]="_getHostElement()"></div>
-<div [class.mat-step-icon]="state !== 'number' || selected"
-     [class.mat-step-icon-not-touched]="state == 'number' && !selected"
```

Comment:
@arodr967 can you add this `mat-step-icon-not-touched` class back for now, there are teams inside Google that are targeting styles at it. You can leave an `@breaking-change` comment to remove it later

Reply:
@mmalerba So simply renaming mat-step-icon-not-selected to mat-step-icon-not-touched would not fix this issue, correct? I would need to bring back mat-step-icon-not-touched to the way it was before?

Reply:
Actually, I think it might be ok.

Reply:
Changes pushed. Let me know if issues have been resolved.

Reply:
I would actually just put the `not-selected` class back how it was and then add back the `not-touched` class with the old logic so that users can continue targeting that class (at least for now)

Reply:
Done.
Also pushed a functionally change. I noticed that when showErrors was set, it show the error on the step initially instead of waiting for the user to interact with the step first.